ui/src/common/ArtistShow.js

```diff
@@ -0,0 +1,388 @@
+        width: '7rem',
+        marginLeft: '0.5rem',
+        flex: '1',
+      },
+    },
+    mbio: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: 'flex',
+        marginLeft: '3%',
+        marginRight: '3%',
+        zIndex: '1',
+      },
+    },
+    content: {
+      flex: '1 0 auto',
+    },
+    cover: {
+      width: 151,
+      boxShadow: '1px 1px 20px 0px #565656',
+      borderRadius: '5px',
+    },
+    artImage: {
+      marginTop: '1rem',
+      marginLeft: '1em',
+      maxHeight: '10rem',
+      backgroundColor: 'inherit',
+      display: 'flex',
+      [theme.breakpoints.down('xs')]: {
+        marginTop: '4rem',
+        maxHeight: '7rem',
+        width: '7rem',
+      },
+    },
+    artDetail: {
+      margin: '1rem',
+      flex: '1',
+      display: 'flex',
+      minHeight: '10rem',
+      [theme.breakpoints.down('xs')]: {
+        display: 'none',
+      },
+    },
+    expand: {
+      display: 'flex',
+      padding: '0',
+      boxShadow: 'none',
+      backgroundColor: 'inherit',
+      fontSize: '0.77rem',
+      color: '#a0a0a0',
+      border: 'none',
+      '& .MuiButton-label': {
+        display: 'contents',
+      },
+      '&.MuiButton-contained:hover': {
+        boxShadow: 'none',
+        backgroundColor: 'inherit !important',
+        color: '#dbdada',
+      },
+    },
+    less: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: ({ link }) => (link ? 'flex' : 'none'),
+        width: '7rem',
+        flex: '1',
+        alignItems: 'flex-end',
+        padding: '0',
+        marginTop: 'auto',
+        border: 'none',
+        boxShadow: '-10px 0px 18px 5px black',
+        background: 'inherit',
+        textTransform: 'capitalize',
+        '&:hover': {
+          background: 'black',
+          boxShadow: '-10px 0px 18px 5px black',
+        },
+        '& .MuiButton-label': {
+          color: `${theme.palette.primary.main}!important`,
+        },
+      },
+    },
+    more: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: ({ link }) => (link ? 'flex' : 'none'),
+        width: '7rem',
+        flex: '1',
+        alignItems: 'flex-start',
+        padding: '0',
+        marginBottom: 'auto',
+        border: 'none',
+        boxShadow: '-10px 0px 18px 5px black',
+        background: 'inherit',
+        textTransform: 'capitalize',
+        '&:hover': {
+          background: 'black',
+          boxShadow: '-10px 0px 18px 5px black',
+        },
+        '& .MuiButton-label': {
+          color: `${theme.palette.primary.main}`,
+        },
+      },
+    },
+    album: {
+      marginBottom: '1em',
+    },
+  }),
+  { name: 'NDArtistPage' }
+)
+
+function ImgMediaCard({ artId, artist }) {
+  const [lastInfo, setlastInfo] = useState()
+  const [expanded, setExpanded] = useState(false)
+
+  const props = { ...artist }
+  const artistProps = props['0']
+  var title = artistProps?.artist
```

Comment:
Please don't use `var`. Always try to use `const`. If not possible, use `let`

Reply:
Sorry, I'll change it asap, and will take care to use `let/const` from next time